.travis.yml

```diff
@@ -31,9 +31,14 @@ install:
  - sudo mkdir -p /var/local/cache/fossology
  - sudo chown $(whoami) /var/local/cache/fossology
 
+sudo: required
+
+services:
+  - docker
+
 before_script:
  - cd src/
- - composer install --dev
```

Comment:
I re-added the `--dev` option. The last test always fails without error log in Travis CI, even though it is completely unrelated to the changes I made